recipes/typos/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.13.10" %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: typos
+  version: {{ version }}
+
+source:
+  url: https://github.com/crate-ci/typos/archive/v{{ version }}.tar.gz
+  sha256: c67df6f7867e72c99187ebcad7c54bb100f4acc7a76fb939f647874b8b0c3ee4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - typos --help
+
+about:
+  home: https://github.com/crate-ci/typos
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
```suggestion
  license_file:
    - LICENSE
    - THIRDPARTY.yml
```

Reply:
There are two license files in the repository:
- https://github.com/crate-ci/typos/blob/master/LICENSE-APACHE
- https://github.com/crate-ci/typos/blob/master/LICENSE-MIT